test/unit/utils.test.js

```diff
@@ -15,12 +15,12 @@ describe('Utilities \'getLabel\'', () => {
 });
 
 describe('Utilities \'formatDate\'', () => {
-  test('reformats a string type date from YYYY-MM-DD to DD/MM/YYYY', () => {
-    expect(formatDate('1987-08-14')).toBe('14/08/1987');
+  test('reformats a string type date from YYYY-MM-DD to Day Month Year', () => {
```

Comment:
Although we don't really use this format, I would recommend keeping it as `DD MMMM YYYY` as it's more common and easier to predict the output.